flutter-for-react-native.md

```diff
@@ -186,16 +188,15 @@ Whereas Dart uses [`Future`](https://www.dartlang.org/tutorials/language/futures
 // Dart
 _getIPAddress() {
   final url = 'https://httpbin.org/ip';
-  Future<HttpRequest> request = HttpRequest.request(url);
-  request.then((value) {
+  HttpRequest.request(url).then((value) {
       print(JSON.decode(value.responseText)['origin']);
   }).catchError((error) => print(error));
 }
 {% endprettify %}
 
 Try it out in [DartPad](https://dartpad.dartlang.org/b68eb981456c5eec03daa3c05ee59486).
 
-**Note:** Check [here](https://www.dartlang.org/tutorials/language/futures) for more details on future.
+**Note:** Check [here](https://www.dartlang.org/tutorials/language/futures) for more details on Futures.
```

Comment:
Linking on "here" is a huge no-no. I'd also put this  into an <aside>, but maybe leave that for me later. But, for now:

For more information, see the [Futures](xxx) documentation.

Reply:
There were a TON of these (linking on "here"). I fixed them all.